datasets/risicozones/risicozones.json

```diff
@@ -0,0 +1,517 @@
+          "afstandseis": {
+            "type": "string",
+            "description": "Gestelde afstand voor veiligheid."
+          },
+          "afstandseisVoldoetInd": {
+            "type": "string",
+            "enum": ["J", "N", "?", "NVT"],
+            "description": "Indicatie of bedrijf voldoet aan gestelde afstand voor veiligheid.",
+            "provenance": "voldoet"
+          }
+        }
+      }
+    },
+    {
+      "id": "lpgtankstation",
+      "title": "LPG tankstation",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
+        "mainGeometry": "geometrie",
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "integer",
+            "description": "Unieke identificatie record."
+          },
+          "stationnummer": {
+            "type": "integer",
+            "description": "Unieke business identificatie record."
+          },
+          "geometrie": {
+            "$ref": "https://geojson.org/schema/Polygon.json",
+            "description": "Vlakgeometrie van een bedrijf."
+          },
+          "geometrie_punt": {
+            "$ref": "https://geojson.org/schema/Point.json",
+            "description": "Locatie van een LPG tankstation."
+          },
+          "dossiernummer": {
+            "type": "string",
+            "description": "Unieknummer dossier die hoort bij LPG tankstation."
+          },
+          "naamBedrijf": {
+            "type": "string",
+            "description": "Bedrijfsnaam.",
+            "provenance": "bedrijfsnaam"
+          },
+          "adresBedrijf": {
+            "type": "string",
+            "description": "Adres van bedrijf.",
+            "provenance": "adres"
+          },
+          "postcodeBedrijf": {
+            "type": "string",
+            "description": "Postcode van bedrijf.",
+            "provenance": "postcode"
+          },
+          "plaatsBedrijf": {
+            "type": "string",
+            "description": "Woonplaats van bedrijf.",
+            "provenance": "plaats"
+          },
+          "stadsdeel": {
+            "type": "string",
+            "description": "Stadsdeel waarin bedrijf valt."
+          },
+          "oliemaatschappij": {
+            "type": "string",
+            "description": "Naam oliemaatschappoj waar bedrijf bij hoort."
+          },
+          "omzet": {
+            "type": "string",
+            "description": "Omzet in m3 per jaar."
+          },
+          "ligging": {
+            "type": "string",
+            "enum": ["Buitengebied", "Industriegebied", "Woongebied"],
+            "description": "Soort ligging waar bedrijf is geplaatst."
+          },
+          "tankaanwezigInd": {
```

Comment:
Is dit bewust ingekort tot "Ind" ipv "Indicatie"? Uitschrijven is wellicht duidelijker voor afnemers van de API?

Reply:
Aangepast, ja beter voluit